plugins/modules/ios_evpn_ethernet.py

```diff
@@ -0,0 +1,1013 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright 2025 Red Hat
+# GNU General Public License v3.0+
+# (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+"""
+The module file for ios_evpn_ethernet
+"""
+
+from __future__ import absolute_import, division, print_function
+
+
+__metaclass__ = type
+
+DOCUMENTATION = """
+module: ios_epvn_ethernet
+short_description: Resource module to configure L2VPN EVPN Ethernet Segment.
+description: This module manages the L2VPN EVPN Ethernet Segment attributes of Cisco IOS network devices.
+version_added: 9.2.0
+author:
+  - Sagar Paul (@KB-perByte)
+  - Jorgen Spange (@jorgenspange)
+notes:
+  - Tested against Cisco IOSXE Version 17.6 on CML.
```

Comment:
I  thought we hadn't tested this against the actual appliance  ?

Reply:
the requester' could test it with 17.12.4 and 17.6.1